stackhpc_monasca_agent_plugins/checks/slurm.py

```diff
@@ -28,7 +28,7 @@
 _SLURM_LIST_JOBS_CMD = ['/usr/bin/scontrol', '-o', 'show', 'job']
 _SLURM_LIST_NODES_CMD = ['/usr/bin/scontrol', '-o', 'show', 'node']
 
-_SLURM_JOB_FIELD_REGEX = ('^JobId=([\d]+)\sJobName=((?i)[\w-]+\.[\w-]+)\sUserI'
+_SLURM_JOB_FIELD_REGEX = ('^JobId=([\d]+)\sJobName=(.*?)\sUserI'
```

Comment:
Isn't `.* == .*?` ?

Reply:
I think `.*` is greedy and `.*?` is lazy? I think lazy is all that's required here.

Reply:
Ah, thanks. Good to know.